api/controllers/UserController.js

```diff
@@ -120,5 +120,22 @@ module.exports = _.mapValues({
     const params = req.allParams();
     Validation.requireParams(params, 'name');
     return res.ok(await Validation.usernameAvailable(params.name));
-  }
+  },
+  async forgotUsername (req, res) {
+    const params = req.allParams();
+    Validation.requireParams(params, 'email');
+    const users = await User.find({email: params.email});
+    res.send(202);
```

Comment:
Can we swap the `res.send(202)` line with the line above it? I'm worried about timing attacks, since right now the response time depends on how long it takes the database to return a list of users, which might be dependent on how many users there are. To avoid this, we should send the response first, and handle everything else afterwards.